types/wlr_export_dmabuf_v1.c

```diff
@@ -150,6 +150,10 @@ static void manager_handle_capture_output(struct wl_client *client,
 			attribs->fd[i], size, attribs->offset[i], attribs->stride[i], i);
 	}
 
+	// Schedule a buffer commit
+	output->needs_frame = true;
```

Comment:
No need for this line, this is already performed by `wlr_output_schedule_frame` now